classes/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] As required for the project, please make sure that `Getters` and `Setters` are present for `@book` and `@person` instance variables. 
Using `getter` and `setter` methods to access and modify the value of a variable provides a controlled and predictable interface for other objects to interact with the variable, while still maintaining the principles of encapsulation and data hiding. i.e.
```ruby
    ...
    attr_accessor :date, :book, :person
   ....
```

Reply:
Updated 🆗